spec/features/user_visits_homepage_spec.rb

```diff
@@ -9,14 +9,14 @@
 
   scenario "successfully redirects to sign in page" do
     visit root_path
-    click_link 'Sign in'
+    click_button "Sign in"
```

Comment:
I think you need to:
1. `git fetch origin/example-home-page-layout` to grab the remote branch
2. `git checkout origin/example-home-page-layout`
3. then checkout a new branch from there